data/saas/config/segment_config.yml

```diff
@@ -7,10 +7,13 @@ saas_config:
   connector_params:
     - name: domain
     - name: personas_domain
+    - name: api_domain
     - name: workspace
     - name: access_token
     - name: namespace_id
     - name: access_secret
+    - name: user_token
```

Comment:
I think I can actually remove these two pieces from the yaml, they are just for creating the user data for testing in the first place.